src/SWP/Bundle/ContentBundle/Resources/config/services.yml

```diff
@@ -107,3 +108,27 @@ services:
         parent: "router.symfony"
         tags:
             - { name: router, priority: 300 }
+
+    swp_content_bundle.valid_route_id:
+        class: SWP\Bundle\ContentBundle\Validator\Constraints\RouteIdValidator
+        arguments:
+            - '@swp.provider.route'
+        tags:
+            - { name: validator.constraint_validator, alias: route_in_database }
+
+    swp_content_bundle.routetoarticle.repository:
```

Comment:
Well, I'm not the first person to have configured stuff in this way. I just followed the pattern that already appears to be there in that file. If we have standard ways of doing things, maybe these should be documented somewhere. At the moment, I can see quite a number of inconsistencies.